packages/frontend/src/redux/actions/account.js

```diff
@@ -410,7 +442,7 @@ export const {
 });
 
 export const checkAndHideLedgerModal = () => async (dispatch, getState) => {
-    const { modal } = getState().ledger;
+    const modal = selectLedgerModal(getState());
     if (modal?.show) {
```

Comment:
Since `modal` can no longer be `undefined` here it might be worth removing the `?`.

Reply:
Nice catch. Done.